tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/ClientCredentialsTests.cs

```diff
@@ -246,30 +248,31 @@ private static IConfidentialClientApplication CreateApp(CredentialType credentia
         private string GetExpectedCacheKey(string clientId, string tenantId)
         {
             return $"{clientId}_{tenantId ?? ""}_AppTokenCache";
-        }       
+        }
 
         internal static long ConvertToTimeT(DateTime time)
         {
             var startTime = new DateTime(1970, 1, 1, 0, 0, 0, 0);
             TimeSpan diff = time - startTime;
             return (long)diff.TotalSeconds;
         }
-
         private static IDictionary<string, string> GetClaims(bool useDefaultClaims = true)
         {
+            const uint JwtToAadLifetimeInSeconds = 60 * 10; // Ten minutes
```

Comment:
any reason to define this here as opposed to using the one in TestConstants? Is the one in TestConstants still being used or can it be removed?

Reply:
Oh, just because we copied this code directly to the docs, so I didn't want a reference to "test consntants"